### PR TITLE
CentOS Stream based sclorg postgres & redis images

### LINKED ISSUE #335: CentOS Stream based sclorg postgres & redis images
Currently, we're using [Centos 7 based postgres & redis images](https://quay.io/organization/centos7).

In https://quay.io/organization/sclorg there are CentOS Stream 8 and 9 based postgres and redis images:

- https://quay.io/repository/sclorg/postgresql-13-c8s (since sclorg/postgresql-container#437)
- https://quay.io/repository/sclorg/postgresql-13-c9s
- https://quay.io/repository/sclorg/redis-6-c8s
- https://quay.io/repository/sclorg/redis-6-c9s

Would be nice to do this together with #327 

EDIT: https://fedoramagazine.org/community-container-images-available-for-applications-development/
EDIT2: https://bugzilla.redhat.com/show_bug.cgi?id=1950420

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes #335

### CODE REVIEW

**Comment by jpopelka** [comment]
Tested (going from PostgreSQL-13.5/gcc-4.8.5/CentOS7 to PostgreSQL-13.7/gcc-11.3.1/C9S) on the cluster and looks OK.

**Comment by jpopelka** [comment]
recheck

File: openshift/postgres.yml.j2 [hunk 1]
```diff
@@ -21,7 +21,10 @@ spec:
         name: postgres-13
     spec:
       containers:
-        - env:
+        - name: postgres
+          image: quay.io/sclorg/postgresql-13-c9s
```
Comment (TomasTomecek) [inline openshift/postgres.yml.j2:25]: nice, we'll bump by 2 major RHEL versions :O

File: openshift/redis.yml [hunk 2]
```diff
@@ -18,20 +18,20 @@ spec:
     spec:
       containers:
         - name: redis
-          image: quay.io/centos7/redis-5-centos7
+          image: quay.io/sclorg/redis-6-c9s
```
Comment (TomasTomecek) [inline openshift/redis.yml:21]: seems that we should be good to go here: upgrading from 5 to 6

https://gitlab.com/gitlab-com/gl-infra/reliability/-/issues/12451
https://github.com/celery/celery/issues/6422

doesn't seem like those issues would affect us

**Review by TomasTomecek — APPROVED** [review]
👏🏼🎉

**Comment by jpopelka** [comment]
deployed to staging instances, will re-deploy production next Monday

**Pull request merged**